js/client-teambuilder.js

```diff
@@ -2835,7 +2835,7 @@
 		},
 		chartClick: function (e) {
 			if (this.search.addFilter(e.currentTarget)) {
-				this.$('input[name=' + this.curChartName + ']').val('').select();
+				this.$('input[name=' + this.curChartName + ']').select();
```

Comment:
This is necessary when searching for filters. i.e. if you want to search for Fire moves, you would type "Fire", click on "Fire", and then you would want "Fire" to no longer be in the move box afterwards.